lesson1/src/arrays/arrayProblems.js

```diff
@@ -21,14 +22,23 @@ class ArrayProblems {
    Eg2: [-3,-1,0,5] -> 5
    */
   static getMax(array) {
-    // todo: do stuff here
+     return Math.max(...array);
   }
 
   /*
    Can you implement getMax again without using any js collection method?
    */
+   /* To be honest, I found this solution from the internet and I still find 
+       it magic it is working. Maybe you can give me more explanation. */
   static getMax2(array) {
-    // todo: 🙌 do magic !
+    let leng = array.length;
+    let max = -Infinity;
```

Comment:
nice ! this looks a *LOT* like the implementation i did; try to walk through it, what happens on an empty array? how many times does your algorithm go through the array?